charts/harvester-load-balancer/values.yaml

```
@@ -8,7 +8,7 @@ image:
   repository: rancher/harvester-load-balancer
   pullPolicy: IfNotPresent
   # Overrides the image tag whose default is the chart appVersion.
-  tag: v0.2.0
+  tag: v0.2.5
```

The image tag defaults to the chart appVersion, so we can drop the explicit version string unless we need something else here.

```suggestion
  tag: ""
```